### PR TITLE
WebRDFS server process, placeholder HTTP handlers

### LINKED ISSUE #129: Create always listening port

### PR DESCRIPTION
**Pull request by LINKIWI**
### Motivation

First step in implementing WebRDFS support. Future developers can build on this skeleton.

### Changeset

* Add `web-rdfs` directory and compile step
* Create `webrdfs` executable, starting HTTP server on a user-specified port
* Add dummy/placeholder handlers for all verbs on the single respected path per the API: https://hadoop.apache.org/docs/r1.0.4/webhdfs.html

### Testing

Manual test:

1. `./build/web-rdfs/webrdfs`
2. `curl localhost:8000/webhdfs/v1/path`
3. Observe HTTP 200 and closed connection

This PR depends on #146, #141.

This closes #129.

### CODE REVIEW

**Comment by heliumj** [comment]
I'm getting:
curl: (7) Failed to connect to localhost port 8000: Connection refused

**Comment by LINKIWI** [comment]
Sorry, typo on my part - the default port is `8080`. But you can specify whatever port you want when you invoke the executable with the `-p` flag.

**Comment by heliumj** [comment]
Ok that fixes it

**Review by vnazz — APPROVED** [review]

File: web-rdfs/include/web_rdfs_server.h [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+   */
+  void register_handler(
+    std::string pattern,
+    const char verb[],
+    std::function<void(
+      std::shared_ptr<HttpServer::Response> response,
+      std::shared_ptr<HttpServer::Request> request)> handler);
+
+
+  std::string getFileInfo(GetFileInfoRequestProto req);
```
Comment (danielbalkum) [inline web-rdfs/include/web_rdfs_server.h:111]: Why return std::string if all functions return ""? Should these perhaps pass along error information from zk?

**Pull request merged**